docs/source/contributing/redux.md

```diff
@@ -39,95 +36,75 @@ export default compose(
-with a `request` key. For a simple example, see
-`src/actions/navigation/navigation.js`. In the `request` key you can set the
-HTTP method type (using the `op` field) and the `path` to the backend. Any
-non-absolute URLs are use the `settings.apiPath` prefix, but you can query any
-other backend server by using a URL that starts with `http://` or `https://`.
-When writing the reducer counterpart, you'll get the backend response available
-as `action.result`.
-
-It's also possible to make multiple backend requests at once, for example to
-batch create content. In that case, set the `action.request` to be a list of
-objects (requests) and consequently, in the reducer, the `action.result` will
-be a list of responses corresponding to each request. See the
-`src/reducers/content` for an example.
-
-In order to make them more generic and allow more reuse, some actions can
-accept a `subrequest` parameter, basically a string that can identify the
-response and "namespace" it in the global state. See for example the `content`
-reducer. Using subrequest is specially important when using the `getContent`
-action, as, without it, it would overwrite the global `state.content` store
-with possible wrong content for the current context.
-
-Creating a "request action" potentially triggers some additional access. For
-example, even if we only declare the `GET_CONTENT` type of action, we can see
-that `GET_CONTENT_SUCCESS`, `GET_CONTENT_PENDING` and `GET_CONTENT_FAIL` are
-also used in the `content` reducer. They are automatically created by the
-special API middleware, available in `src/middleware/api.js`.
+Backend network fetches are automatically triggered by creating a Redux action with a `request` key.
+For a simple example, see {file}`src/actions/navigation/navigation.js`.
+In the `request` key you can set the HTTP method type (using the `op` field) and the `path` to the backend.
+Any non-absolute URLs are using the `settings.apiPath` prefix, but you can query any other backend server by using a URL that starts with `http://` or `https://`.
+When writing the reducer counterpart, you'll get the backend response available as `action.result`.
+
+It's also possible to make multiple backend requests at once, for example to batch create content.
+In that case, set the `action.request` to be a list of objects (requests) and consequently, in the reducer, the `action.result` will be a list of responses corresponding to each request.
+See the {file}`src/reducers/content` for an example.
+
+In order to make them more generic and allow more reuse, some actions can accept a `subrequest` parameter, basically a string that can identify the response and "namespace" it in the global state.
+See for example the `content` reducer.
+Using subrequest is specially important when using the `getContent` action, as, without it, it would overwrite the global `state.content` store with possible wrong content for the current context.
```

Comment:
```suggestion
Using `subrequest` is especially important when using the `getContent` action, as without it, it would overwrite the global `state.content` store with possibly wrong content for the current context.
```